mkdocs_redirects/plugin.py

```diff
@@ -54,7 +54,8 @@ def write_html(site_dir, old_path, new_path):
 def get_relative_html_path(old_page, new_page, use_directory_urls):
     """ Return the relative path from the old html path to the new html path"""
     old_path = get_html_path(old_page, use_directory_urls)
-    new_path = get_html_path(new_page, use_directory_urls)
+    new_path_with_hash = get_html_path(new_page, use_directory_urls)
+    new_path, new_hash_fragment = _split_hash_fragment(new_path_with_hash)
```

Comment:
Why not just put these two lines in the opposite order and then `get_html_path` doesn't need to also have the splitting logic